poutyne/framework/metrics/epoch_metrics.py

```diff
@@ -0,0 +1,139 @@
+import sys
+
+from abc import ABC, abstractmethod
+
+try:
+    import allennlp.training.metrics.FBetaMeasure as FBetaMeasure
+except ImportError:
+    FBetaMeasure = None
+
+
+class EpochMetric(ABC):
+    # pylint: disable=line-too-long
+    """
+    The abstract class representing a epoch metric which can be accumulated at each batch and calculated at the end
+    of the epoch.
+    """
+
+    @abstractmethod
+    def __call__(self, y_prediction, y_true):
+        """
+        To define the behavior of the metric when called.
+
+        Args:
+            y_prediction: The prediction of the model.
+            y_true: Target to evaluate the model.
+        """
+        pass
+
+    @abstractmethod
+    def __init__(self):
+        self.__name__ = self.__class__.__name__
+
+    @abstractmethod
+    def get_metric(self, reset):
+        """
+        Compute and return the metric. Optionally also reset the running measure.
+        """
+        pass
+
+
+class FBeta(EpochMetric):
+    """
+    Wrapper around the Allen NLP FBetaMeasure class.
+
+    Args:
+        beta : (float), optional (default = 1.0) The strength of recall versus precision in the F-score.
```

Comment:
```suggestion
        beta : (float, optional) The strength of recall versus precision in the F-score.
                  (Default value = 1.0).
```

Coherence with other documentation.